root/etc/cb/integrations/cb-defense-syslog/cb-defense-syslog.conf.example

```diff
@@ -104,25 +104,35 @@ https_ssl_verify=True
 [cbdefense1]
 
 #
-# Cb Defense Connector ID
+# Cb Defense API Connector ID
 #
-connector_id = F8KF111111
+api_connector_id = GO5M9531A4
```

Comment:
I am hoping these aren't functional API keys right?

Reply:
No I made fake ones for the config file. I was using another config file for testing.

Reply:
If you would like to verify that they are fake you are more than willing to do so.

Reply:
Thanks! just wanted to make sure